packages/filebrowser/src/crumbs.ts

```diff
@@ -64,6 +73,11 @@ export class BreadCrumbs extends Widget {
     this.addClass(BREADCRUMB_CLASS);
     this._crumbs = Private.createCrumbs();
     this._crumbSeps = Private.createCrumbSeparators();
+    this._hasPreferred =
+      PageConfig.getOption('preferredPath') !== '/' ? true : false;
```

Comment:
Shouldn't this compared to an empty string, in case `preferredPath` is not set?

Reply:
Otherwise it seems to be assuming that `preferredPath` should be part of the page config.

Reply:
This seems to be where the default `/` is set, for applications using the `LabHandler`:

https://github.com/jupyterlab/jupyterlab_server/blob/93703e2242788c6b8083b351e25ec8113930420d/jupyterlab_server/handlers.py#L89-L99